dvc/remote/ssh/__init__.py

```diff
@@ -278,5 +280,5 @@ def exists_with_progress(chunks):
                 results = executor.map(exists_with_progress, chunks)
                 in_remote = itertools.chain.from_iterable(results)
                 ret = list(itertools.compress(checksums, in_remote))
-                pbar.update_desc("", 0)  # clear path name description
+                pbar.set_description_str("Querying remote cache")
```

Comment:
Can we avoid this repetition? Just return to starting desc in the end, maybe even automatically.

Reply:
@casperdcl Great points! I totally agree, let's remove those.

Reply:
ok now (intermediate bars are removed):

```bash
$ pip install -e git+https://github.com/tqdm/tqdm.git@devel#egg=tqdm
$ git clone https://github.com/casperdcl/dvc.git --branch ui-tidy
$ cd dvc/
$ pip install .
$ dvc pull
100%|██████████|Checkout                          3/3 [00:00<00:00,   791file/s]
```